### PR TITLE
don't save submission if one with the same tweetId already exists in airtable

### PR DESCRIPTION
**Pull request by JosiasAurel**
These changes includes a few things.
- Will prevent saving a submission if one with the same tweetId already exists
- Deleted duplicate submissions in the airtable
- Will poll for duplicates and report if any are found

### CODE REVIEW

**Review by grymmy — CHANGES_REQUESTED** [review]
Can we instead just make it impossible to create the duplicate entries in the first place?

I'm being a bit facetious saying this, but this is like turning on the sprinkler system 30 minutes after the fire already burned the building down...

File: app/persistence.py [hunk 1]
```diff
@@ -39,9 +39,11 @@ def queue_work(self, tweetId, twitterHandle, puzzleId, expression):
         :param expression: The sinerider graph expression in the submission
         """
         print("queueing: %s %s %s %s" % (tweetId, twitterHandle, puzzleId, expression))
-        self.work_queue_table.create(
-            {"tweetId": tweetId, "twitterHandle": twitterHandle, "puzzleId": puzzleId, "expression": expression,
-             "completed": False, "attempts": 0})
+        existing_submission = self.get_one_row(self.work_queue_table, "tweetId", tweetId)
+        if existing_submission is None:
```
Comment (grymmy) [inline app/persistence.py:44]: Should we be handling the else case here?

File: app/persistence.py [hunk 2]
```diff
@@ -100,16 +104,6 @@ def increment_attempts_queued_work(self, tweet_id):
         self.work_queue_table.update(row["id"], {"attempts": attempts})
         return attempts
 
-    def increment_attempts_queued_work(self, tweet_id):
```
Comment (grymmy) [inline app/persistence.py:103]: was deleting this method intentional?

**Review by grymmy — APPROVED** [review]
Per our discussion, approving this diff, please make the fix to reconstitute the deleted increment_attempts_queued_work method before landing.

File: app/persistence.py (hunk 2 above)
Comment (JosiasAurel) [reply app/persistence.py:103]: not intentional... fixing!

**Pull request merged**